.gitignore

```diff
@@ -7,3 +7,15 @@ node_modules
 .DS_Store
 .idea
 storybook-static
+
+
+# Android/IJ
+#
+.classpath
+.cxx
+.gradle
+.idea
+.project
+.settings
+local.properties
+android.iml
```

Comment:
We can probably get rid of these too as it is related to the android folders

Reply:
Sure thing

Reply:
@rodrigoeidelvein removed in e5e79cf007f4e92caea078782dafd9a29233114a